components/brave_rewards/resources/donate/components/siteBanner.tsx

```diff
@@ -147,6 +147,7 @@ class Banner extends React.Component<Props, State> {
         currentAmount={this.state.currentAmount}
         onClose={this.onClose}
         social={this.generateSocialLinks()}
+        addFundsLink={'brave://rewards/#add-funds'}
```

Comment:
nit: can we get this from a function like `get addFundsLink`? Functionally it is the same I just think it's a bit cleaner :)

Reply:
done